src/Evolution/Executables/Cce/CharacteristicExtract.hpp

```diff
@@ -32,11 +32,17 @@
 struct EvolutionMetavars {
   using system = Cce::System;
 
+  static constexpr bool uses_inverse_coordinates = false;
```

Comment:
Maybe `uses_inertial_cartesian_coordinates` ?  Or `uses_partially_flat_cartesian_coordinates` ?   It would be nice if the name here is consistent with the names of the coordinates.